pkg/resources/cluster/patches/csi-controller-disable-volume-group-snapshots.yaml

```diff
@@ -11,6 +11,10 @@
       template:
         spec:
           containers:
+          - name: linstor-csi
```

Comment:
Add a comment to the yaml with the explanation instead of just having it in the commit history?